agr_api/src/main/java/org/alliancegenome/api/service/ESService.java

```diff
@@ -95,7 +95,7 @@ private void generateFilter(BoolQueryBuilder bool, String filterName, String fil
 			//Log.info("Or Filter: " + filterName + " " + filterValue);
 			BoolQueryBuilder orClause = boolQuery();
 			String[] elements = filterValue.split("\\|");
-			Arrays.stream(elements).forEach(element -> orClause.should(QueryBuilders.termQuery(filterName, escapeValue(element))));
+			Arrays.stream(elements).forEach(element -> orClause.should(QueryBuilders.termQuery(filterName, element)));
```

Comment:
@cmpich and I had a conversation about this long ago, and I thought it would come back to bite us... maybe this is it?

Reply:
Escaping the characters breaks the OR keyword searches in the terms contain brackets.  We don't escape the characters for single term searches so I don't see the need to do it for OR